src/java/org/apache/cassandra/db/compaction/CompactionTask.java

```diff
@@ -129,10 +129,10 @@ protected void runMayThrow() throws Exception
 
         final long expectedWriteSize = checkAvailableDiskSpaceAndGetWriteSize(checkAvailableDiskSpaceFunction);
 
-        if (expectedWriteSize > 5 * 1024 * 1024 * 1024)
+        if (expectedWriteSize > 5 * 1024 * 1024 * 1024L)
         {
```

Comment:
can you make this magic number a `private static final` constant, explicitly as a `long`